src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -95,10 +95,8 @@ def arch_repo_table_cell(repo, arch, package_name)
     result += "#{theclass}"
     result +=" buildstatus'>".html_safe
 
-    if %w(unresolvable blocked).include?(code)
+    if %w(- unresolvable blocked excluded scheduled).include?(code)
       result += link_to(code, '#', title: link_title, id: status_id, class: code)
-    elsif %w(- excluded scheduled).include?(code)
-      result += code
```

Comment:
Before:
``
<td class="status_excluded buildstatus">excluded</td>
``

Now:
``
<td class="status_excluded buildstatus"><a href="#" ...>excluded</a></td>
``

I don't get your commit message... Without the <a> tag the CSS doesn't apply? Maybe we should fix than the CSS instead of wrapping an <a> tag around it without needing it?